src/ProductCatalogService.Application/Commands/Models/Product.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ProductCatalogService.Application.Commands.Models
+{
+    /// <summary>
+    /// Model for Product
+    /// </summary>
+    public class Product
```

Comment:
Move this class into a common folder for use by GetProduct and PostSearchProduct